packages/suite/src/hooks/wallet/coinmarket/useCoinmarketWatchTrade.ts

```diff
@@ -80,6 +80,11 @@ const coinmarketWatchTrade = async <T extends CoinmarketTradeType>({
                 error: exchangeResponse.error,
             };
 
+            if (exchangeResponse.sendAddress) {
+                tradeData.sendAddress = exchangeResponse.sendAddress;
+                tradeData.partnerPaymentExtraId = exchangeResponse.partnerPaymentExtraId;
+            }
```

Comment:
This comes from invity-api schema, renaming it would require maintaining backward compatibility

Reply:
But it has not yet been defined, or am I mistaken? You will create invity-api so you can define it in the same way as a sell.

Reply:
`tradeData.sendAddress` is already defined in invity-api. We can change the naming in WatchExchangeTradeResponse, but imho it would be weird to use different names for the same field.

Reply:
Ok. So let it be then. :)